### PR TITLE
[WIP] Create Transactions Model and Views

### PR DESCRIPTION
**Pull request by jeleencubillas**
## The Story
- User Stories require a record of transactions between buyers and brokers
- We created  a transaction model with foreign keys for user_id, stock_id, and broker_id
- This PR also integrates a bug fix for the Admin model, which was set to rememberable but no remembered_at is in the schema
- This PR also integrates preliminary CSS styling

## Screenshots
preliminary-views
Preliminary CSS Styling

## Commit Messages
- Create Transactions Model and Views

## Whats Missing
- [ ] Database Associations
- [ ] Refine Styling

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery with: :exception
```
Comment (analyn34) [inline app/controllers/application_controller.rb:2]: why did you remove this one?

File: app/views/users/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<h1>Edit User</h1>
+
+<%= render "form", user: @user %>
```
Comment (analyn34) [inline app/views/users/edit.html.erb:3]: add new line at the end of file

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -1,11 +1,61 @@
...
+  end
+
+  # def configure_sign_up_params
+  #   devise_parameter_sanitizer.permit(:create, keys: [:first_name, :last_name, :date_of_birth, :role, :approved])
+  # end
+  
+  # def configure_account_update_params
+  #   devise_parameter_sanitizer.permit(:update, keys: [:first_name, :last_name, :date_of_birth, :role, :approved])
+  # end
+
```
Comment (analyn34) [inline app/controllers/users_controller.rb:60]: remove the unnecessary comments

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -1,11 +1,61 @@
 class UsersController < ApplicationController
   
+  # before_action :configure_sign_up_params, :configure_sign_up_params
```
Comment (analyn34) [inline app/controllers/users_controller.rb:3]: remove the unnecessary comments

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (jeleencubillas) [reply app/controllers/application_controller.rb:2]: Sorry Maam was debugging, will add it again

**Review by analyn34 — APPROVED** [review]
